controller/userController.js

```diff
@@ -0,0 +1,152 @@
+const userService = require('../services/userService');
+const middleNick = require('../util/nickMaker');
+const middleJwt = require('../middleware/jwt');
+
+const signUp = async(req,res) => {
+    try{
+        if(!req.body.email || !req.body.password || !req.body.phone_number || !req.body.birthday){
+            return res.status(400).json({message : 'KEY_ERROR'})
+        }
+        const userEmail = req.body.email
+        const userPassword = req.body.password
+        const userNickname = req.body.nickname          || middleNick.nicknameMaker()
+        const userPhoneNumber = req.body.phone_number
+        const userBirthDay = req.body.birthday
+        const userGender = req.body.gender              || ""
```

Comment:
```suggestion
      const { 
        email: userEmail, 
        password: userPassword, 
        nickname: userNickname = middleNick.nicknameMaker(),
        phone_number: userPhoneNumber,
        birthday: userBirthday,
        gender: userGender = ""
      } = req.body
```

해당 문법이 같은 역할을 합니다 :) 더 깔끔하게 이용할 수 있을 것 같습니다